### PR TITLE
Release: version update to 1.7.0

### PR DESCRIPTION
**Pull request by ipainchaud**
I have also updated the term of service function name.

### CODE REVIEW

File: lib/src/platform_client.dart
Comment (ipainchaud) [inline lib/src/platform_client.dart:1]: in order, the things I despise the most of this world, after pure evil of course are:

1. the "hide known extensions" option of Windows
2. poison ivy
3. auto-formatters which format what you haven't changed

I'm sorry for all those auto-formatter changes.

File: lib/src/platform_client.dart [hunk 1]
```diff
@@ -538,7 +524,7 @@ class PlatformClient {
 
   /// Updates the user's terms of service and returns
   /// the logged-in [User].
-  Future<User> updateTermsOfService(bool tosAccepted) async {
+  Future<User> updateTermsOfServiceAccepted(bool tosAccepted) async {
```
Comment (ipainchaud) [inline lib/src/platform_client.dart:527]: behold, the only real change of this file

File: lib/src/platform_client.dart
Comment (vsomayaji) [reply lib/src/platform_client.dart:1]: Have we aligned on a line length? We were initially using `120` but I think @chimon2000 prefers `80`. If we go with `80`, I'm guessing most of these changes will get reverted.

Comment (chimon2000) [reply lib/src/platform_client.dart:1]: We should update the VSCode settings to be 120, I added that setting to the Flutter repository but apparently missed it here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.7.0] - 2023-09-27
+### New
+- `PlatformClient.updateTermsOfServiceAccepted(bool)` to update the Terms of Service status (#91)
+- `Room.onConnectionFailed` to update the Room user on any SFU connection failure (#92)
+- `SFUConnection.isConnectionFailed` to be able to identify which SFUConnection has failed (#92)
```
Comment (vsomayaji) [inline CHANGELOG.md:11]: Did we add `SFUConnection.isConnectionFailed` or is this out of date?

**Review by vsomayaji — COMMENTED** [review]
I recommend changing the target branch from `fix/SFUFailIssue` to `main`.

File: CHANGELOG.md (hunk 2 above)
Comment (ipainchaud) [reply CHANGELOG.md:11]: sorry! This was removed afterward and I forgot to remove it here.

File: lib/src/sfu_connection.dart [hunk 3]
```diff
@@ -30,6 +30,9 @@ class SfuConnection {
   late final RTCPeerConnection _peerConnection;
   late final RTCRtpTransceiver _audio;
   late final RTCRtpTransceiver _video;
+  RTCPeerConnectionState? _connectionState;
+
+  bool get isConnectionFailed => _connectionState == RTCPeerConnectionState.RTCPeerConnectionStateFailed;
```
Comment (vsomayaji) [inline lib/src/sfu_connection.dart:35]: Is this needed?

**Review by vsomayaji — APPROVED** [review]

**Review by vsomayaji — APPROVED** [review]

**Pull request merged**